ProjectReunion.props

```diff
@@ -11,13 +11,24 @@
     <!-- If not set, default it to no . -->
     <ProjectReunionBuildPipeline>0</ProjectReunionBuildPipeline>
   </PropertyGroup>
+  <Target Name="DirectoryBuildPropsInfo">
+    <Message Importance="High" Text="ProjectReunion.props detects ProjectReunionBuildPipeline=$(ProjectReunionBuildPipeline)"/>
+  </Target>
 
   <!-- Conditional behavior for build pipeline vs local development -->
```

Comment:
What is this file used for? It doesn't seem to be used by local VS development forcing me to add to Directory.Build.props but obviously used in pipeline builds else it bad code caused errors. The comment on line 34 seems to reinforce this.

Yes the override info's duplicated right now. There was more in this file I wasn't sure of why or if it's safe for local development so worst case we've got some duplicate information we can refactor later. If this file's purely for pipeline builds then it should be renamed something more specific.

Regardless, the changes unblock critical path work. This cleanup can be done at a later date

Reply:
@mikenelte This would be a good question for Michael who was in the original V team to bring up most of this. I've never touch this file before.